src/components/apps/listing/Listing.js

```diff
@@ -196,8 +196,13 @@ function Listing(props) {
     }, [selected]);
 
     useEffect(() => {
-        setAddToBagEnabled(selected && selected.length > 0);
-    }, [selected]);
+        const selApps = getSelectedApps();
+        setAddToBagEnabled(
+            selApps &&
+                selected.length > 0 &&
+                selApps?.filter((app) => app.is_public === true).length === 0
```

Comment:
No need to explicitly check `=== true`:
```suggestion
                selApps?.filter((app) => app.is_public).length === 0
```
Or slightly shorter:
```suggestion
                !selApps?.find((app) => app.is_public)
```